core/modules/system/config/system.core.json

```diff
@@ -46,6 +46,7 @@
     "maintenance_mode_message": "[site:name] is currently under maintenance. We should be back shortly. Thank you for your patience.",
     "maintenance_page_maximum_age": 10,
     "maintenance_theme": null,
+    "menu_breakpoint": "48em",
```

Comment:
I can't find any update hook regarding that setting. Existing sites wouldn't have any breakpoint setting then - unless the admin sets one, right?

Or maybe a wrapper function with a fallback value of "48em" could solve that, in case config_get returned NULL.

Reply:
Done, thanks!